assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -126,38 +135,53 @@ const CartLineItemRow = ( { lineItem = {} } ) => {
 		catalogVisibility === 'hidden' || catalogVisibility === 'search';
 
 	// Allow extensions to filter how the price is displayed. Ie: prepending or appending some values.
-	const productPriceFormat = __experimentalApplyCheckoutFilter( {
-		filterName: 'cartItemPrice',
-		defaultValue: '<price/>',
-		arg: {
-			cartItem: lineItem,
-			block: 'cart',
-		},
-		validation: ( value ) =>
-			typeof value === 'string' && value.includes( '<price/>' ),
-	} );
+	const productPriceFormat = useMemo(
+		() =>
+			__experimentalApplyCheckoutFilter( {
+				filterName: 'cartItemPrice',
+				defaultValue: '<price/>',
+				extensions,
+				arg: {
+					cartItem: lineItem,
+					context: 'cart',
+				},
+				validation: ( value ) =>
+					mustBeString( value ) && mustContain( value, '<price/>' ),
+			} ),
+		[ lineItem, extensions ]
+	);
 
-	const subtotalPriceFormat = __experimentalApplyCheckoutFilter( {
-		filterName: 'subtotalPriceFormat',
-		defaultValue: '<price/>',
-		arg: {
-			lineItem,
-		},
-		// Only accept strings.
-		validation: ( value ) =>
-			typeof value === 'string' && value.includes( '<price/>' ),
-	} );
+	const subtotalPriceFormat = useMemo(
+		() =>
+			__experimentalApplyCheckoutFilter( {
+				filterName: 'subtotalPriceFormat',
+				defaultValue: '<price/>',
+				extensions,
+				arg: {
+					lineItem,
```

Comment:
If we're going to pass the context in the arg for one filter (`cartItemPrice`), then we should ensure we do it for all filters imo. Even if the filter only gets applied in one place, we may eventually do it elsewhere, so it would be good to have this as a standardised parameter we pass through.

Reply:
Applying the same filter elsewhere could be a breaking change so I don't see value in prematurely passing in context when we only have the filter in once price.

I did update our context calls for several filters but not all filters (skipped TotalsFilter because it's a single component so we can't really know the context).